drivers/SmartThings/zigbee-thermostat/fingerprints.yml

```diff
@@ -84,10 +84,15 @@ zigbeeManufacturer:
     manufacturer: Stelpro
     model: ST218
     deviceProfileName: thermostat-temperature-temperaturealarm
-  - id: "Stelpro/MaestroStat"
-    deviceLabel: Stelpro Thermostat
+  - id: "Maestro Thermostat"
+    deviceLabel: Maestro Thermostat
     manufacturer: Stelpro
-    model: MaestroStat
+    model: SMT402AD
+    deviceProfileName: thermostat-stelpro-profile
```

Comment:
Hello @gcavalcantistelpro Thank you for your submission. I would encourage that we focus this change only on adding one new device, the SAT402ZB device. Can you please update the PR to no longer make changes to the "Selpro/MaestroStat"device?

Reply:
Hello @lelandblue , I've just added a commit for the SAT402ZB thermostat, i left the maestro untouched and added new files for this specific thermostat

Reply:
Hello @lelandblue , i could validate locally that the device is working after my last commit, please let me know if any further modifications are required on the PR side. Thanks